braintree.go

```diff
@@ -103,7 +103,7 @@ func (g *Braintree) executeVersion(method, path string, xmlObj interface{}, v ap
 	if err != nil {
 		return nil, err
 	}
-	defer resp.Body.Close()
+	defer func() { _ = resp.Body.Close() }()
```

Comment:
Explicitly ignoring the return error of the close on the response body, because we don't care if it fails because we're reading, not writing. We will have already fully read the body at the point we call close.